Makefile

```diff
@@ -2,16 +2,19 @@ RAKE = docker-compose run app bundle exec rake
 RUN = docker-compose run app
 
 install:
-	@make secert
+	@make secret
 	@touch app.local.env
 	@$(RUN) bundle install --retry=3 --jobs=2
+	@echo "\n**Notice**: Wait for the initialization of DB...\n"
+	@echo "...\n"
+	@sleep 10
```

Comment:
And the time of `bundle install` is different everyone, so we may consider it "immediately" better.